style.css

```diff
@@ -0,0 +1,39 @@
+main {
```

Comment:
Don't use position absolute and relative for all element's position. It is not good for performance